public/app/core/reducers/root.ts

```diff
@@ -4,6 +4,7 @@ import sharedReducers from 'app/core/reducers';
 import ldapReducers from 'app/features/admin/state/reducers';
 import alertingReducers from 'app/features/alerting/state/reducers';
 import apiKeysReducers from 'app/features/api-keys/state/reducers';
+import { pubDashApi } from 'app/features/dashboard/api/pubDashApi';
```

Comment:
nit: remove braces

Reply:
this is not a default export, so we need to use braces